libs/blocks/quiz/quiz.js

```diff
@@ -10,21 +10,20 @@ import {
 } from './utils.js';
 import StepIndicator from './stepIndicator.js';
 
-async function loadFragments(fragmentURL) {
+export async function loadFragments(fragmentURL, createFragment) {
```

Comment:
you pass createFragment as parameter, I think it is not good as import it, and also you update here and not udpate the calling one line 285, that will cause issue

Reply:
I think you can see the issue that Jacky foresaw by looking at your code in the context of Elaine's data that loads the footer fragment. In the uar-integration branch, with her data, you can see "personalize your website experience" at the bottom of the content area when you navigate through the first question. 

https://uar-integration--milo--adobecom.hlx.page/drafts/borges/end-to-end/?q-category=photo

In your branch, though, the fragment url is still unresolved in the footer div. 

https://quiz-tests--milo--fullcolorcoder.hlx.page/drafts/borges/end-to-end/?q-category=photo

`<div class="quiz-footer"><a href="https://main--milo--adobecom.hlx.page/drafts/borges/end-to-end/fragments/footer-questionfour"></a></div>`